src/herder/TransactionQueue.cpp

```diff
@@ -225,6 +239,7 @@ TransactionQueue::shift()
     {
         mSizeByAge[i]->set_count(sizes[i]);
     }
+    recomputeQueueSizeOps();
```

Comment:
You don't need to do this. We're already computing `sizes` which is the number of pending transactions of different ages. Let's just do something similar for a new variable `sizeOps` (it doesn't have to separate by age). This should lead to less code overall and make it easier to read.

Reply:
yeah I wanted to avoid the extra tracking, but it's not that bad as we do it for fees anyways